### DISCUSSION TITLE
Feature request: support for Discussions in GitHub project boards

### DISCUSSION THREAD

**Discussion by erikr** [community/community#3238] [Discussions]
Our team, and many others, use GitHub Projects for agile development task management.

It would be great to be able to add Discussions as cards, just like Issues.

This is mentioned in https://github.com/github/feedback/discussions/41#discussioncomment-269831 but the topic is distinct.

I searched but did not see a duplicate Discussion for this feature request already, but if one exists, please let me know!

Thanks.

**Comment by erikr** [comment]
@becca @evi-liu Is this on the road map, or has anyone else requested this? Just curious.

**Reply by evi-liu** [reply]
I'm curious if you could share more about the value of having Discussions on your project board? Generally, we view project boards to be used for executable pieces of work, such as Issues.

**Reply by erikr** [reply]
Certainly! We would like Discussions to be on our Project Board because they are executable units of collaborative work, albeit less "defined in a way that translates into code changes" than Issues.

Discussions can block other Issues or Discussions, e.g. before executing on a data modeling strategy for X analysis plan, we need to achieve consensus around Y topic via an asynchronous Discussion.

Currently, since Discussions are on their own page from Issues, and not Board-able, they are less visible. Our team members bookmark the Project Board (or Issues page) as their task list when they prioritize what to work on that day or sprint. Moreover at Standup we (now virtually) gather around Project Boards.

Because of above, it is helpful for Discussions to have visibility on the board. Moreover, having a UX with one URL : one Discussion : one card on the board forces us to cohere discussion topics into more granular units, which is good.

Glad to provide other info if helpful.

**Reply by munderseth** [reply]
I would agree with @erikr.  We would leverage Discussions (assuming available in a private repo) for `Exploratory Testing` - providing observations while testing. 

With `script-based testing`, generating Issues is appropriate. Managing Issues (i.e. bugs) on a Project Board works well. 

But with `Exploratory Testing`, questions, comments, observations, etc., are generated.  Discussions map well to Exploratory Testing - which of course might have a different life-cycle than a bug-based Issue. But a Discussions still can require management to resolve. 

Thus, a Board seems to me to be a very nice fit.  Boards for `Notes`, `Issues`, and `Discussions` 😄

**Comment by Stvad** [comment]
Would love see this too. I'm using Discussions to collect feature requests & would be interested in using Project to present a public roadmap based on those requests.  

---
One workaround I thought of is creating cards that just have shortlink to the discussion like #3238. But it's not great as shortlink is only rendered as number and you have to hover over it to see the content. And it involves a bunch of manual actions still
